source/end_sixtrack.f90

```diff
@@ -129,8 +128,8 @@ subroutine abend(endMsg)
   if(napxo == 0) napxo = napx
   write(crlog,"(2(a,i0))") "ABEND_CR> Writing fort.10, lines ",napxo,"/",napx
   flush(crlog)
-  do j=1,napxo ! Must the dummy file really be napxo times the same dummy line?
-    write(10,"(a)",iostat=ierro) outLine(1:60*26)
+  do j=1,napxo,2 ! Must the dummy file really be napxo times the same dummy line?
```

Comment:
Since it was writing napxo lines, and not napxo/2, I doubt there is anything that actually checks the number of records is correct anyway. Otherwise it would already have failed. But as we discussed earlier today, this is a dangerous way of returning fort.10 to BOINC as it will generate a file that will always pass the comparison check!

Reply:
Yeah, but at least this doesn't make it *more* dangerous.

Anyway, that's why I tagged the others. I'll make an issue.

Reply:
#856